.github/workflows/openhands-resolver.yml

```diff
@@ -278,6 +278,12 @@ jobs:
               console.error('Error reading results file:', error);
             }
 
+            if (logContent.includes("Updated pull request")) {
+              console.log("Updated pull request found. Skipping comment.");
+              return;
+            }
+
+
             try {
               if (success) {
```

Comment:
In that case, it would be nice to comment on that, if possible. There's a lot of cases here, tbh, I'm not sure including it in the conditions below would be clearer...

Reply:
Good points! I'll clean this up a little